### PR TITLE
Split ICCProfile into several files

### PR DESCRIPTION
**Pull request by nico**
As requested on discord. No behavior change.

(TagTypes.cpp will grow more after this.)

### CODE REVIEW

File: Userland/Libraries/LibGfx/ICC/Profile.cpp [hunk 1]
```diff
@@ -1588,9 +1219,9 @@ Crypto::Hash::MD5::DigestType Profile::compute_id(ReadonlyBytes bytes)
     const u8 zero[16] = {};
     Crypto::Hash::MD5 md5;
     md5.update(bytes.slice(0, 44));
-    md5.update(ReadonlyBytes { zero, 4 }); // profile flags field
+    md5.update(ReadonlyBytes { zero, 4 });  // profile flags field
```
Comment (linusg) [inline Userland/Libraries/LibGfx/ICC/Profile.cpp:1222]: linter is not happy about this whitespace change

Comment (nico) [reply Userland/Libraries/LibGfx/ICC/Profile.cpp:1222]: different clang-format versions here than remotely, i suppose :/

Comment (linusg) [reply Userland/Libraries/LibGfx/ICC/Profile.cpp]: `Ubuntu clang-format version 15.0.7` on CI, which generally tries to use the latest stable version that is not a total pain to install on ubuntu - if you're using something newer which matches this criteria, please lmk!

**Comment by nico** [comment]
Looks like it lints now.

**Review by linusg — APPROVED** [review]
Thanks!

**Comment by linusg** [comment]
`FuzzICCProfile.cpp` needs an update too, apparently :)

**Pull request merged**